### DISCUSSION TITLE
OBS-MIDI don't save parameters when I restart OBS

### DISCUSSION THREAD

**Discussion by spencerpablo** [cpyarger/obs-midi#146] [Q&A]
Hi friends,

i have a stranger problem with OBS-MIDI. I enable my keyboard option, put all parameters for my MIDI controller for change scenes. And work very well. But when I close OBS Studio and Open again all configuration in OBS-MIDI disappear. I Tried reinstall everything but don't worked. Somebody can help me?

macOS Catalina 10.15.7
OBS Studio 27.0.1
Obs-MIDI 0.9.3-Alpha-3x

Thanks!
![Captura de Tela 2021-08-17 às 13 00 11](https://user-images.githubusercontent.com/89092796/129760637-c2740608-4f0f-4c6e-9cde-a63284fe3d96.png)
![Captura de Tela 2021-08-17 às 13 00 46](https://user-images.githubusercontent.com/89092796/129760648-1cecd585-95b9-44ef-ac72-cabba77898ce.png)

**Comment by cpyarger** [comment]
With this information I have no clue honestly
